<launch>
    <node pkg="pointcloud_labeling_tools" exec="label_points_by_cuboids" name="sparse_vegetation_labeler">
<!--        <remap from="input_pointcloud" to="/hugin_raf_1/radar_data"/>-->
        <remap from="input_pointcloud" to="/point_cloud_deskewed"/>
        <param name="world_frame_id" value="map"/>
        <param name="cuboid_file" value="$(find-pkg-share pointcloud_labeling_tools)/config/viking_hill_labels.json"/>
        <param name="sample_id" value="0"/>
        <param name="label_set_name" value="ground-truth"/>
        <param name="label_priorities" value="3, 7, 1, 5, 4, 6, 2" value-sep=", "/>

        <param name="use_sim_time" value="true"/>
    </node>
</launch>